.github/workflows/kubeconform-validation.yml

```
@@ -0,0 +1,47 @@
+name: Kubeconform Validation
+
+on:
+  pull_request:
+
+jobs:
+  load-environments:
+    name: Load environments
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: echo "::set-output name=matrix::$(ls .github/environments/ | jq -R -s -c 'split("\n")[:-1]')"
+  validation:
+    needs: load-environments
+    strategy:
+      matrix:
+        environment: ${{fromJson(needs.load-environments.outputs.matrix)}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
```

```suggestion
          python-version: 3.8
```
I don't think Tutor was compatible with Python 3.12 in this version